src/common/transformers/CAPITALIZE.js

```diff
@@ -0,0 +1,31 @@
+export const capitalizeString = value => {
+    if (!value) {
```

Comment:
Be it capitalize, uniq, or trim, the only thing that change is the function applied to the value.
Please refactor in order to reuse the common logic.

eg:

```js
const transform = fn => value => {
    if(!value) {
        return null;
    }
     ...
    return fn(value);
}
```

Reply:
This way you only have to test transform, and fn, and this will avoid all test copy paste.